tests/fxl-orientation_duplicate/OPS/package.opf

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-09-28</dc:date>
+    <dc:description>EPUB creators must not declare the orienation property more than once.</dc:description>
```

Comment:
I guess this is our first intentional core-spec test. Should this say something like "EPUBCheck should produce an error."?

Reply:
Also I think the description should be in this format: "rendition:orientation is specified twice. EPUBCheck should produce an error."

Reply:
Also, note typo: "orienation"